app/demo-loop/Transporter.cc

```diff
@@ -185,6 +185,8 @@ TransporterResult Transporter<M>::operator()(SpanConstPrimary primaries)
     StepperInput input;
     input.params = input_.params;
     input.num_track_slots = input_.num_track_slots;
+    // TODO: change when doing multithreading on the front end
+    input.stream_id = StreamId{0};
     input.sync = input_.sync;
     Stepper<M> step(std::move(input));
```

Comment:
I suppose Philippe's comment could apply here as well
```suggestion
    // TODO: change StreamId{0} when doing multithreading on the front end
    Stepper<M> step(
        {input_.params, StreamId{0}, input_.num_track_slots, input_.sync});
```

Reply:
Yes, this could save a handful of inexpensive CPU instructions that are called once per code execution 🙃 but I think the reduced code clarity doesn't compensate.

Looks like avoid the move saves two instructions out of ~50, a `mov` and an `punpcklqdq`: https://godbolt.org/z/9PozcGYza so definitely not worth it IMHO.